index.html

```diff
@@ -25,12 +31,15 @@ <h1>Analytic Hierarchy Process (AHP) Tool</h1>
       <aside>
         <h3>New to the AHP?</h3>
         <p>Check out <a href="https://en.wikipedia.org/wiki/Analytic_hierarchy_process" target="_blank">Wikipedia's article on it</a>.</p>
+        <h3>Data Management</h3>
+        <p><a onclick="resetForm()">RESET</a> - <a onclick="exportForm()">EXPORT</a> - <a onclick="importForm()">IMPORT</a></p>
```

Comment:
You can style these to look like links, but semantically please make them buttons:
```suggestion
        <p><button onclick="resetForm()">Reset</button> - <button onclick="exportForm()">Export</button> - <button onclick="importForm()">Import</button></p>
```
If you want the text to be ALL-CAPS, you can use `text-transform: uppercase;` instead of actually using all capital letters